File: WatchApp/SuggestionItem.m
//
//  SuggestionItem.m
//  WatchApp
//
//  Created by Stephane Bisson on 1/27/15.
//  Copyright (c) 2015 Stephane Bisson. All rights reserved.
//

#import "SuggestionItem.h"

@implementation SuggestionItem

+ (SuggestionItem*) withText:(NSString*)text {
    SuggestionItem *s = [[SuggestionItem alloc] init];
    s.completionText = text;
    return s;
}

@end

File: WatchApp/SuggestionItem.swift

import Foundation
import TRAutocompleteView

@objc class SuggestionItem : NSObject, TRSuggestionItem {
    
    var suggestionText:String
    
    init (completionText:String) {
        self.suggestionText = completionText
    }
    
    class func withText(text:String) -> SuggestionItem {
        var s:SuggestionItem = SuggestionItem(completionText:text)
        return s
    }
    
    func completionText() -> String! {
        return self.suggestionText
    }
    
}
